File: tests/fixtures/positive/security/JdbcStatementExecute.java
package test;

import java.sql.Connection;
import java.sql.Statement;

class JdbcStatementExecuteJavaFixture {
    void load(Connection connection, String id) throws Exception {
        Statement stmt = connection.createStatement();
        stmt.executeQuery("SELECT * FROM users WHERE id = " + id);
    }
}

File: tests/fixtures/positive/security/JdbcStatementExecute.kt
package test

import java.sql.Connection

class JdbcStatementExecuteFixture {
    fun load(connection: Connection, id: String) {
        connection.createStatement().executeQuery("SELECT * FROM users WHERE id = $id")
        val stmt = connection.createStatement()
        stmt.executeUpdate("DELETE FROM users WHERE id = " + id)
    }
}
